snippets/dart_snippets/pubspec.lock

```diff
@@ -44,12 +44,10 @@ packages:
   breez_liquid:
     dependency: "direct main"
     description:
-      path: "."
-      ref: HEAD
-      resolved-ref: "3111663fbc91aa918a9b44f7deadaf97f94462b9"
-      url: "https://github.com/breez/breez-sdk-liquid-dart"
-    source: git
-    version: "0.7.2"
+      path: "packages/breez-sdk-liquid-dart"
+      relative: true
+    source: path
+    version: "0.6.0-dev1"
```

Comment:
This from local testing?

Reply:
Oops, thanks. Dropped the changed to the file in [647517a](https://github.com/breez/breez-sdk-liquid-docs/pull/86/commits/647517a41a8e15349d62ab8fd339542e6c4e2abe)